projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+# School events
+
+School events are events that are specifically organized and intented for:
+
+* students (e.g. a school performance)
+* teachers (e.g. an after-school training)
+
+School events are published on [Cultuurkuur](https://www.cultuurkuur.be), but not on UiTinVlaanderen.
+
+> If you’re planning to create school events in UiTdatabank with the Entry API, please contact <content.cultuurkuur@publiq.be> in advance so that we can guide you through this process.
```

Comment:
What does this guidance include that is not included in this documentation? Or is it more that we want to have an idea who is going to implement this and they need to contact us so we know that they are going to implement this?

Because it seems like this documentation covers everything, so it's not unlikely that someone is going to implement it like it documented here without contacting us. So if you want to be sure they contact us for whatever reason, I think we need to clarify here that we will either provide extra info that is not provided here and that they should have (although I'm not sure why we cannot document it here then), OR they need to contact us to notify us that they are going to implement this so we can do some kind of review etc.

Alternatively if you meant that they can optionally contact us about any questions they might have related to school events, I'd rephrase it to clarify that.

Reply:
Good question / remark! From what I remember the main reason we want integrators to contact us is because we want to make sure they only start implementing this if it's the right choice cost-benefit wise. Many cultural organisations already have an integration with our Entry API for their public events. They often only have a few school events each year, which they enter via www.uitdatabank.be. 

For these organisations it's probably more cost effective to keep entering their school events via www.uitdatabank.be instead of developing & maintaining the very custom integration for their school events (with all the custom logic of the location in consultation with the school, complex use of the labels (that change from time to time), etc.)

An integration with the entry api for school events is only cost effective for very big organisations I think, or when you can share the cost of the integration with other customers / partners (like the partners of Parcours vzw).

Long story short: I'll make rephrase the copy / warning and let Eva have a double check on this.

Reply:
Okay that definitely makes a lot of sense. If you can rephrase this info block to mention it like that I think we will have a lot more success in actually making them contact us to discuss that first before they implement it.